frontend/database/modules/datasets.js

```diff
@@ -444,6 +444,10 @@ const getters = {
 };
 
 const actions = {
+  async getRubrixVersion() {
+    const response = await this.$axios.$get("_info");
+    return response.rubrix_version;
+  },
```

Comment:
Could we move this action to a separate module since has no relation to datasets?